src/toolbar_button.js

```diff
@@ -28,6 +28,15 @@ function updateToolbarButton(tab) {
   };
 
   chrome.storage.local.get('options', function(items){
+    if (
+      'dismiss_news_badge' in items['options'] &&
+      items['options']['dismiss_news_badge']
+    ) {
+      chrome.browserAction.setBadgeText({ text: '' });
+    } else {
+      chrome.browserAction.setBadgeText({ text: '📣' });
```

Comment:
Everything looks good in this PR, but on my OS, the badge was a deep red color, which made it look like an error. I also wasn't quite able to figure out what the megaphone icon was without looking really closely. 

So two thoughts:

1. Can we change the background color to something that looks less like an error? Maybe blue? 

2. Can we use a more common emoji? I think the :gift: one is pretty common and what Slack uses?

Reply:
> Can we change the background color to something that looks less like an error? Maybe blue?

We can customize the background color but I don't think we can tweak the style of the emoji. Here's an example:

![image](https://github.com/user-attachments/assets/2116c4e1-bdb1-403d-9061-166f234e1de9)

![image](https://github.com/user-attachments/assets/9da36b32-3540-494e-801f-cc32abc31ef4)

> Can we use a more common emoji? 

I suggest using the bell emoji 🔔:  I just noticed the megaphone emoji renders as a red megaphone in Gmail, but the bell emoji 🔔 works fine.

![image](https://github.com/user-attachments/assets/fb006856-d019-4cac-aec2-68f322ca8aeb)

Reply:
Great. Let's :shipit: to browsers!

Reply:
👍 I'll update the emoji